apps/segment/.eslintrc.cjs

```diff
@@ -3,6 +3,11 @@ module.exports = {
   extends: ["@saleor/eslint-config-apps"],
   plugins: ["@typescript-eslint", "neverthrow", "node"],
   rules: {
+    "no-console": "error",
+    "@saleor/saleor-app/logger-leak": "error",
+    "max-params": ["error", { max: 3 }],
+    "@vitest/prefer-strict-equal": "error",
+    "@vitest/prefer-vi-mocked": "error",
```

Comment:
why its not shared with eslint-plugin-saleor?

Reply:
I want to migrate app by app - after I merge all PRs with lint issues fixes I will move those rules to `eslint-plugin-saleor` 😄